### PR TITLE
[WIP] add more logging

### PR DESCRIPTION
**Pull request by jwkaltz**
Using name for all loggers. Adding some log statements.

### CODE REVIEW

**Comment by jwkaltz** [comment]
@sbrunner I would like to add more logging. Does is make sense for you like this?
I can then add some more.

File: pyramid_oereb/lib/readers/municipality.py [hunk 1]
```diff
@@ -38,5 +41,7 @@ def read(self):
             The list of all found records. Since these are not filtered by any criteria the list simply
             contains all records delivered by the source.
         """
+        log.debug("read()")
```
Comment (ger-benjamin) [inline pyramid_oereb/lib/readers/municipality.py]: Are these kind of log really useful ?

Comment (jwkaltz) [reply pyramid_oereb/lib/readers/municipality.py]: Regarding PR #97 : you mean closing the PR 97 and instead I will suggest logging entries in error cases within this PR (302)? Yes, that would be ok for me.

Regarding your question "really useful": this one I guess probably not... except to have a trace (in debug Level) to understand where a request went through. What do you think about the traces with parameter values, would these be (more) useful?

**Comment by jwkaltz** [comment]
Removing - interesting parts of this PR have now been addressed in other PRs.

**Pull request closed without merging**